crates/chia-sdk-driver/src/primitives/clawback.rs

```diff
@@ -22,6 +42,74 @@ pub struct Clawback {
 }
 
 impl Clawback {
+    pub fn parse_children(
```

Comment:
I think it would be good to fuzz parsers for all puzzles, to make sure there's no way to trigger UB or panic